automated-comments/go/two-fer/replace_stub_comments.md

```diff
@@ -0,0 +1,5 @@
+The stub comments should be replaced with proper comments on package and function.
+Accurate, helpful comments on how to use the code, are very important to Go developers. 
+This is a great guide to writing good comments: [Effective Go: Commentary](https://golang.org/doc/effective_go.html#commentary).
+The `godoc` tool will produce documentation from the code comments automatically.
```

Comment:
I suggest removing the "but how" question from this comment:

```suggestion
The `godoc` tool produces documentation from the code comments automatically, when executed.
```

Or something similar.